specification/datamigration/resource-manager/readme.md

```diff
@@ -92,6 +92,25 @@ input-file:
 - Microsoft.DataMigration/preview/2018-03-31-preview/definitions/MigrationValidation.json
 - Microsoft.DataMigration/preview/2018-03-31-preview/definitions/ValidateMigrationInputSqlServerSqlMITask.json
 ```
+
```

Comment:
Line 31 in readme.md has default as "tag: package-2018-03-31-preview".
Change this to the GA version. The public repo sdk picks up the version number from here.

Reply:
Thanks, done.